be/src/exprs/expr.cpp

```diff
@@ -735,19 +735,13 @@ void Expr::get_jit_exprs(std::vector<Expr*>& exprs) {
 // This method searches from top to bottom for compilable expressions.
 // Once a compilable expression is found, it skips over its compilable subexpressions and continues the search downwards.
 Status Expr::replace_compilable_exprs(Expr** expr, ObjectPool* pool) {
-    std::vector<Expr*> next_exprs;
     if ((*expr)->should_compile()) {
         // If the current expression is compilable, we will replace it with a JITExpr.
         // This expression and its compilable subexpressions will be compiled into a single function.
         *expr = JITExpr::create(pool, *expr);
-        // Skip over the compilable subexpressions.
-        (*expr)->get_uncompilable_exprs(next_exprs);
-    } else {
-        // Search for compilable expressions in the subexpressions.
-        next_exprs = _children;
```

Comment:
why delete this? I don't get the point

Reply:
If this section of code is not deleted, when replacing ﻿expr, if the replaced node is not at the top level, it cannot truly affect the ﻿expr tree. This issue was introduced when supporting more complex compilable ﻿expr tree rules.